settings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dictionaries": {
+    "build-in": ["en_US", "en_GB"],
+    "custom": ["assets/dic/extra-words.dic"]
```

Comment:
> a custom dictionary has been created in `assets/dic/extra-words.dic`

@brianamarie, I think we should spell out folder names, `assets/dictionary/extra-words.dic`